tests/devices/test_default_qubit.py

```diff
@@ -2273,6 +2269,7 @@ def test_diagonal_operation_case(self, inverse, mocker, monkeypatch):
             assert np.allclose(res_mat, np.diag(op.matrix))
             assert np.allclose(res_wires, wires)
 
+    @pytest.mark.parametrize("inverse", [True, False])
```

Comment:
Why is the `.inv()` method being removed in the earlier test, but not these next two?

Reply:
good point!

Reply:
I removed it from the following one for the same reason, but the last does not seem to test diagonal gates. We can clean this up when looking at `inv`